components/module-components/Perk-Cards/PerkCards.tsx

```diff
@@ -0,0 +1,116 @@
.0687 17.8717 36.5038 16.8357 35.7 16H44ZM24.5 0C25.4193 -1.93718e-08 26.3295 0.18106 27.1788 0.532843C28.0281 0.884626 28.7997 1.40024 29.4497 2.05025C30.0998 2.70026 30.6154 3.47194 30.9672 4.32122C31.3189 5.17049 31.5 6.08075 31.5 7C31.5 7.91925 31.3189 8.8295 30.9672 9.67878C30.6154 10.5281 30.0998 11.2997 29.4497 11.9497C28.7997 12.5998 28.0281 13.1154 27.1788 13.4672C26.3295 13.8189 25.4193 14 24.5 14C22.6435 14 20.863 13.2625 19.5503 11.9497C18.2375 10.637 17.5 8.85652 17.5 7C17.5 5.14348 18.2375 3.36301 19.5503 2.05025C20.863 0.737498 22.6435 3.91231e-08 24.5 0ZM40.506 2C41.2939 2 42.0741 2.15519 42.8021 2.45672C43.5301 2.75825 44.1915 3.20021 44.7486 3.75736C45.3058 4.31451 45.7477 4.97595 46.0493 5.7039C46.3508 6.43185 46.506 7.21207 46.506 8C46.506 8.78793 46.3508 9.56815 46.0493 10.2961C45.7477 11.0241 45.3058 11.6855 44.7486 12.2426C44.1915 12.7998 43.5301 13.2417 42.8021 13.5433C42.0741 13.8448 41.2939 14 40.506 14C38.9147 14 37.3886 13.3679 36.2634 12.2426C35.1381 11.1174 34.506 9.5913 34.506 8C34.506 6.4087 35.1381 4.88258 36.2634 3.75736C37.3886 2.63214 38.9147 2 40.506 2ZM8.494 2C9.28193 2 10.0621 2.15519 10.7901 2.45672C11.5181 2.75825 12.1795 3.20021 12.7366 3.75736C13.2938 4.31451 13.7358 4.97595 14.0373 5.7039C14.3388 6.43185 14.494 7.21207 14.494 8C14.494 8.78793 14.3388 9.56815 14.0373 10.2961C13.7358 11.0241 13.2938 11.6855 12.7366 12.2426C12.1795 12.7998 11.5181 13.2417 10.7901 13.5433C10.0621 13.8448 9.28193 14 8.494 14C6.9027 14 5.37658 13.3679 4.25136 12.2426C3.12614 11.1174 2.494 9.5913 2.494 8C2.494 6.4087 3.12614 4.88258 4.25136 3.75736C5.37658 2.63214 6.9027 2 8.494 2ZM24.5 3C23.4391 3 22.4217 3.42143 21.6716 4.17157C20.9214 4.92172 20.5 5.93913 20.5 7C20.5 8.06087 20.9214 9.07828 21.6716 9.82843C22.4217 10.5786 23.4391 11 24.5 11C25.5609 11 26.5783 10.5786 27.3284 9.82843C28.0786 9.07828 28.5 8.06087 28.5 7C28.5 5.93913 28.0786 4.92172 27.3284 4.17157C26.5783 3.42143 25.5609 3 24.5 3ZM40.506 5C40.112 5 39.7219 5.0776 39.3579 5.22836C38.994 5.37913 38.6633 5.6001 38.3847 5.87868C38.1061 6.15726 37.8851 6.48797 37.7344 6.85195C37.5836 7.21593 37.506 7.60603 37.506 8C37.506 8.39397 37.5836 8.78407 37.7344 9.14805C37.8851 9.51203 38.1061 9.84274 38.3847 10.1213C38.6633 10.3999 38.994 10.6209 39.3579 10.7716C39.7219 10.9224 40.112 11 40.506 11C41.3017 11 42.0647 10.6839 42.6273 10.1213C43.1899 9.55871 43.506 8.79565 43.506 8C43.506 7.20435 43.1899 6.44129 42.6273 5.87868C42.0647 5.31607 41.3017 5 40.506 5ZM8.494 5C8.10004 5 7.70993 5.0776 7.34595 5.22836C6.98197 5.37913 6.65126 5.6001 6.37268 5.87868C6.0941 6.15726 5.87313 6.48797 5.72236 6.85195C5.5716 7.21593 5.494 7.60603 5.494 8C5.494 8.39397 5.5716 8.78407 5.72236 9.14805C5.87313 9.51203 6.0941 9.84274 6.37268 10.1213C6.65126 10.3999 6.98197 10.6209 7.34595 10.7716C7.70993 10.9224 8.10004 11 8.494 11C9.28965 11 10.0527 10.6839 10.6153 10.1213C11.1779 9.55871 11.494 8.79565 11.494 8C11.494 7.20435 11.1779 6.44129 10.6153 5.87868C10.0527 5.31607 9.28965 5 8.494 5Z"
```

Comment:
Better approach here would be to extract this data into a file called `perkCardsData.ts`
After you have done that, you can rename this component something like PerkCardList, and you would iterate over the data from `perkCardsData.ts` and show PerksCard for each.
This way we will have one place where we have the data, and we can add/remove/edit. And another place where you would have the whole layout with the PerkCards